src/Standards/Generic/Sniffs/CodeAnalysis/EmptyStatementSniff.php

```diff
@@ -74,10 +81,16 @@ public function process(File $phpcsFile, $stackPtr)
             return;
         }
 
+        if ($this->allowComments === true) {
+            $emptyTokens = ([T_WHITESPACE => T_WHITESPACE] + Tokens::$phpcsCommentTokens);
+        } else {
+            $emptyTokens = Tokens::$emptyTokens;
+        }
+
         $next = $phpcsFile->findNext(
-            Tokens::$emptyTokens,
+            $emptyTokens,
             ($token['scope_opener'] + 1),
-            ($token['scope_closer'] - 1),
+            $token['scope_closer'],
```

Comment:
Wasn't expecting to have to change this, but skips the last token otherwise.

Reply:
This is a known issue, see https://github.com/squizlabs/PHP_CodeSniffer/issues/1319